website/src/css/custom.css

```
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * Any CSS included here will be global. The classic template
+ * bundles Infima by default. Infima is a CSS framework designed to
+ * work well for content-centric websites.
+ */
+
+/* You can override the default Infima variables here. */
+:root {
+  --ifm-color-primary: #ef4242;
+  --ifm-color-primary-dark: #ed2626;
+  --ifm-color-primary-darker: #eb1818;
+  --ifm-color-primary-darkest: #c51111;
+  --ifm-color-primary-light: #f15e5e;
+  --ifm-color-primary-lighter: #f36c6c;
+  --ifm-color-primary-lightest: #f69696;
+}
```

The current default code font size is really small.
```suggestion
:root {
  --ifm-color-primary: #ef4242;
  --ifm-color-primary-dark: #ed2626;
  --ifm-color-primary-darker: #eb1818;
  --ifm-color-primary-darkest: #c51111;
  --ifm-color-primary-light: #f15e5e;
  --ifm-color-primary-lighter: #f36c6c;
  --ifm-color-primary-lightest: #f69696;
  --ifm-code-font-size: 95%;
}
```